ceph/ceph_admin/bootstrap.py

```diff
@@ -194,7 +196,9 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
         elif custom_repo:
             self.set_tool_repo(repo=custom_repo)
         else:
-            self.set_tool_repo()
+            repos = ["Tools"]
+            for node in self.cluster.get_nodes():
+                setup_repos(node, base_url=base_url, repos=repos)
```

Comment:
Is this something that we can do in a separate PR?

Reply:
This pr needs this changes as we are seeing failure with gpgcheck
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-2GYFRW/Cephadm-ansible_clients_playbook_0.log

Reply:
"yum-config-manager --setopt=gpgcheck=0 --add-repo <url>" cannot be used in a single command and has to be split into two
1.yum-config-manager --add-repo <url>
2. yum-config-manager --setopt=<name>.gpgcheck=0
As  set_tool_repo() is already existing method it also needs modification